stories/create-stories-from-doc.js

```diff
@@ -49,7 +49,14 @@ const files = [
   ['Spinners', '../_site/docs/5.0/components/spinners/index.html'],
   ['SteppedProcess', '../_site/docs/5.0/components/stepped-process/index.html'],
   ['Toasts', '../_site/docs/5.0/components/toasts/index.html'],
-  ['Tooltips', '../_site/docs/5.0/components/tooltips/index.html'],
+  [
+    'Tooltips',
+    '../_site/docs/5.0/components/tooltips/index.html',
+    'var tooltipTriggerList = [].slice.call(document.querySelectorAll(\'[data-bs-toggle="tooltip"]\'));\
+    var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {\
```

Comment:
This is my first idea to make components that use JavaScript work in Storybook: adding the specific JavaScript part at the end of the `.stories.js` file.
Another approach would be to add ALL the JavaScript specific parts within the global `preview-head.html` file; but it means to load unused things for nothing and also to have specific content separated from the automatic generator of stories.

Reply:
I'm struggling to find a good architecture for all of this!
I'm wondering if I shouldn't use/embed _site/assets/application.js_ and the CSS in _site/assets/scss_: it would fix the JavaScript issues automatically (I think) and the specific rendering issues of the modals for example.
But if I do that, I'll need to add a `<div class="bd-example bd-example-*">` around the code in Storybook that will be displayed when the user will click on the "Code" tab...

Reply:
Note: The designers embed the "Docs" part and the JavaScript is not executed there...